flutter_kernel_transformers/lib/track_widget_constructor_locations.dart

```diff
@@ -0,0 +1,516 @@
+library track_widget_constructor_locations;
```

Comment:
needs license/copyright block like other dart files

Reply:
done. also added one to server.dart which was missing one.